central/pod/datastore/datastore_impl_real_test.go

```diff
@@ -0,0 +1,172 @@
+//go:build sql_integration
+
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	podSearch "github.com/stackrox/rox/central/pod/datastore/internal/search"
+	podStore "github.com/stackrox/rox/central/pod/store/postgres"
+	processIndicatorDataStore "github.com/stackrox/rox/central/processindicator/datastore"
+	processIndicatorSearch "github.com/stackrox/rox/central/processindicator/search"
+	processIndicatorStorage "github.com/stackrox/rox/central/processindicator/store/postgres"
+	plopDataStore "github.com/stackrox/rox/central/processlisteningonport/datastore"
+	plopStore "github.com/stackrox/rox/central/processlisteningonport/store"
+	plopPostgresStore "github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/fixtures/fixtureconsts"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/process/filter"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/sac/testutils"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestPodDatastore(t *testing.T) {
+	suite.Run(t, new(PodDatastoreSuite))
+}
+
+type PodDatastoreSuite struct {
+	suite.Suite
+
+	datastore          *datastoreImpl
+	indicatorDataStore processIndicatorDataStore.DataStore
+	plopDS             plopDataStore.DataStore
+	plopStorage        plopStore.Store
```

Comment:
This looks like it is only used in `SetupTest` so it can be local to `SetupTest`